indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -97,6 +97,15 @@
                 <li>{{ item }}</li>
             {%- endfor %}
         </ul>
+    {% elif field.input_type == 'sessions' and friendly_data%}
```

Comment:
I'm a bit unsure about how to apply this since reglist and pdfs are prepared to show the data in one row and the optimal for this

In the reglist, I decided to show only the session blocks since I believe the date is not neccesary, also, I think is not enough to separate the session blocks with commas since there's a lot of text and I feel it's not clear where ends and starts each session block. 

In addition, in the PDF files, I belive I could show the days since that format allows much more information, but I dislike the format and I didn't find a good way of showing it.

However, I also think I should keep the same format as per MultiCheckbox. 

I show some pictures of what I have in hands right now to accept suggestions.

`Reglist`
<img width="996" alt="Screenshot 2024-03-27 at 11 06 50" src="https://github.com/indico/indico/assets/11741165/576365bd-3f56-4812-8c38-13e4c790c1ac">


`PDF Book`
<img width="704" alt="Screenshot 2024-03-27 at 11 09 27" src="https://github.com/indico/indico/assets/11741165/1665f0d0-f586-451d-8bdd-36cb5d133804">